.travis.yml

```diff
@@ -34,12 +34,10 @@ rvm:
 services:
   - redis-server
 
-bundler_args: --without development production --retry=3 --jobs=16
-
 install:
   - nvm install
   - npm install -g yarn
-  - bundle install
+  - bundle install --path=vendor/bundle --without development production --retry=3 --jobs=16
```

Comment:
The bundler caching docs claim that when `cache.bundler` is turned on in Travis config, that it will use `vendor/bundle` by default - https://docs.travis-ci.com/user/caching/#Bundler - does that not happen?